devbox.go

```diff
@@ -109,30 +106,36 @@ func (d *Devbox) Plan() (*plansdk.Plan, error) {
 	return plansdk.MergeUserPlan(userPlan, automatedPlan)
 }
 
+// Plan creates a plan of the actions that devbox will take to generate its
+// shell environment.
+func (d *Devbox) BuildPlan() (*plansdk.Plan, error) {
+	userPlan := d.convertToPlan()
+	buildPlan, err := planner.FindBuildablePlan(d.srcDir)
```

Comment:
What do you think of the API if we were to rename as:
- `planner.GetPlan` -> `planner.GetShellPlan`
- `planner.FindBuildablePlan` -> `planner.GetBuildPlan`

This would make it analogous to `Devbox.ShellPlan()` (if you were to rename it from current `Devbox.Plan()` and `Devbox.BuildPlan()`

Reply:
I'll do `planner.FindBuildablePlan` -> `planner.GetBuildPlan`

Didn't change `GetPlan` for the same reason as noted above